src/main/resources/react4xp/_entries/PubArchiveStatCalendarLinks.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { Link } from '@statisticsnorway/ssb-component-library'
+import PropTypes from 'prop-types'
+import { ArrowRight, Calendar } from 'react-feather'
+
+const PubArchiveStatCalendarLinks = (props) => {
+  return (
+    <React.Fragment>
+      <div className="container">
+        <div className="row justify-content-md-center justify-content-start">
+          <div className="col-12 col-md-6 links-wrapper">
+            {props.PublicationLink ? <Link
+              href={props.PublicationLink}
+              linkType="profiled"
+              className='publication-link mr-5 mb-md-0 mb-4'
+              icon={<ArrowRight size="18" />}
+            >
+              <span
```

Comment:
her trenger man vel ikke å bruke span med dangerouslySetInnerHTML. Skal vel holde med {props.PublicationText} rett i link elementet, samme for CalendarLink